org-code-javabuilder/playground/src/test/java/org/code/playground/ImageItemTest.java

```diff
@@ -28,7 +28,8 @@ public void setUp() {
         "",
         "",
         "",
-        mock(JavabuilderFileManager.class));
+        mock(JavabuilderFileManager.class),
+        null);
```

Comment:
Idea for later - might be worth making a TestGlobalProtocol that can be constructed using a builder so we don't have to update all these tests whenever we make a change to `GlobalProtocol.create()`?

Reply:
that's a good idea, I can pick that up as a follow-up

Reply:
I'm going to leave it as-is for now because I think the best way to do this would be to have some TestUtils package that has helpers for our tests, which feels too heavy for this specific situation. Otherwise we would need to import a test class into other packages which isn't a great pattern. For now it's not too onerous to update this in multiple places.